use a download icon atop the column of download controls

Alex-Jordan opened the pull request:
From the instructor view in the problem sets page, the column of checkboxes for download has no indication of its purpose (unless you read the dowload button that is down and to the left and figure it out). This puts an icon with tooltip text atop that column, which helps and does not take up as much space as a word would take up. The same icon gets placed there for student view, which I think is harmless.

Also I shortened "Download PDF or TeX Hardcopy for Selected Sets" to just "Download Hardcopy for Selected Sets". That button is very wide.

pstaabp commented:
I might suggest that the button (and tooltip) is "Generate Hardcopy for Select Sets", since clicking the button actually takes to the generate page, not download.

Alex-Jordan commented:
I changed the button to say "Generate Hardcopy for Select Sets". I changed the tooltip to just say "Generate Hardcopy" because it is the same icon and tooltip that students see, and for them there are no "selected sets". They can only do one at a time.

pstaabp reviewed (approved):
Looks good now.

mgage commented:
@drgrice1 we'll need to rebuild the .po files after this pull request.  The labels on some buttons have been changed.

I'd like to make a detailed description of how to scrape the code and find out which strings need to be changed in the .pot file.  Can you help with this -- my experience is out of date. If we can automate this even better.  --- Mike

drgrice1 commented:
It is all completely automated already.  Just run the script update_localization_files.
